Gemfile

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+group :development do
+  gem "rubocop-discourse"
+  gem "syntax_tree", github: "ruby-syntax-tree/syntax_tree", branch: "main"
```

Comment:
pinning to main since the gem is a dev dependency and isn't version bumped very often, and its fix to get `plugin/disable_auto_ternary` was merged in after the last version.